### DISCUSSION TITLE
Support for the Intel PRO/1000 MT Desktop from VirtualBox(?)

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#2916] [Ideas]
Is it prohibited to merge the support for the Intel Pro/1000 MT desktop network card from VirtualBox?

**Comment by dhrdlicka** [comment]
No, it's not prohibited.

**Comment by rigred** [comment]
Pro 1000 is also emulated under various open source projects (libvirt/qemu) so I don't see a problem with it even from legal aspects.

**Comment by jadasse** [comment]
oh ok

**Comment by jadasse** [comment]
Hopefully that becomes a feature...
